os/arch/arm/src/armv7-a/arm_assert.c

```diff
@@ -595,22 +569,18 @@ void up_assert(const uint8_t *filename, int lineno)
 	}
 
 	irqstate_t flags = irqsave();
-	lldbg_noarg("\n\n");
-	lldbg_noarg("##########################################################################################################################################\n");
-	lldbg_noarg("Assertion failed\n");
-	lldbg_noarg("##########################################################################################################################################\n");
+	lldbg("Assertion failed\n");
 #ifdef CONFIG_SECURITY_LEVEL
 	lldbg_noarg("security level: %d\n", get_security_level());
 #endif
 	/* Print assert detail information and dump state,
 	 * but if os security level is high, It is not printed.
 	 */
 	if (!IS_SECURE_STATE()) {
+		lldbg_noarg("===========================================================Assertion_Details==========================================\n");
 		print_assert_detail(filename, lineno, fault_tcb, asserted_location);
 	}
 
-	lldbg_noarg("##########################################################################################################################################\n");
```

Comment:
I think there should be a closing log line.
Because we need to check if the log output is interrupted without completion.

Reply:
Yeah agreed, then we would also need to add closing log line to os/arch/arm/src/armv8-m/up_assert.c.

Reply:
yes, Please add closing line to all arch code.

Reply:
done.. added.